README.md

```diff
@@ -469,6 +513,49 @@ Would handle request such as `GET /users` with the following query: `select u fr
 
 Supplying `constVal` implicitly sets `defaultVal` to empty.
 
+Support for [SpEL](https://docs.spring.io/spring/docs/5.2.7.RELEASE/spring-framework-reference/core.html#expressions) expression and [property placeholders]((https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/support/PropertySourcesPlaceholderConfigurer.html)) in `defaultVal` could be enabled in following way:
```

Comment:
does the description differ from constVal? If not, then perhaps just mention that SPEL is configured the same way as constVal?